modules/odc-connecting-components.adoc

```diff
@@ -51,21 +54,21 @@ features, see https://access.redhat.com/support/offerings/techpreview/.
 ====
 
 
-
 [NOTE]
 ====
-Currently, only a few specific Operators like the *etcd* and the *PostgresSQL Database* Operator's service instances are bindable.
+Currently, a few specific Operators like the *etcd* and the *PostgresSQL Database* Operator's service instances are bindable.
 ====
 
-To connect the Node.js component using a binding connector:
+You can establish a binding connection with Operator-backed components.
+
+This procedure walks through an example of creating a binding connection between a PostgreSQL Database service and a Node.js application. To create a binding connection with a service that is backed by the PostgreSQL Database Operator, you need to first add the Red Hat provided PostgreSQL Database Operator to the *OperatorHub* using a backing `OperatorSource`, and then install the Operator.
```

Comment:
This is still missing the suggested changes of:

* need to -> must
* Red Hat provided -> Red Hat-provided

Reply:
Fixed, thank you. Sorry for missing that one.

Reply:
No worries, thanks for the update!